packages/aeproject-cli/aeproject-fire-editor/fire-editor.js

```diff
@@ -155,6 +159,51 @@ const installModuleDependencies = async () => {
     }
 }
 
+const isNodeVersionSupported = (fireEditorNodeVersion = constants.FIRE_EDITOR_NODE_VERSION, localNodeVersion = process.version) => {
+    let fireEditorTokens = fireEditorNodeVersion.split('.');
+    let localNodeTokens = localNodeVersion.replace('v', '').split('.');
+
+    fireEditorTokens = fireEditorTokens.map(Number);
+    localNodeTokens = localNodeTokens.map(Number);
+
+    function isValidPart (x) {
+        return (/^\d+$/).test(x);
+    }
+
+    if (!fireEditorTokens.every(isValidPart) || !localNodeTokens.every(isValidPart)) {
+        return NaN;
+    }
+
+    for (var i = 0; i < fireEditorTokens.length; ++i) {
+
+        if (fireEditorTokens[i] == localNodeTokens[i]) {
+            continue;
+        } else if (localNodeTokens[i] > fireEditorTokens[i]) {
+            return 1;
+        } else {
+            return -1;
+        }
+    }
+
+    if (fireEditorTokens.length != localNodeTokens.length) {
```

Comment:
what whill hapen if local version is 10.13?

Reply:
fixed it